Open Judge System/Web/OJS.Web/Areas/Administration/Views/Contests/ChangeActiveParticipantsEndTime.cshtml

```diff
@@ -178,17 +212,40 @@
                         });
                 });
 
+            $('.change-time-method-checkbox').change(function () {
+                $('div.change-time-method-container').hide();
+                $('.change-time-method-checkbox').prop('checked', false);
+                $(this).parents().first().children('div.change-time-method-container').show();
+                $(this).prop('checked', true);
+            });
+
+            function setConfirmWindowHeader() {
+                var confirmWindowHeader = $('#confirm-window-header');
+                var headerPrefix = '@Resource.Confirmation_message_header_prefix ';
+                var headerText = '';
+
+                if ($('#change-time-by-interval').is(':visible')) {
+                    headerText = '@Resource.Confirmation_message_header_change_by_interval';
+                } else if ($('#change-time-by-user').is(':visible')) {
+                    headerText = '@Resource.Confirmation_message_header_change_by_user';
+                }
+
+                var parser = new DOMParser;
+                var header = parser.parseFromString(headerPrefix + headerText, 'text/html');
+
+                confirmWindowHeader.html(header.body.textContent);
+            }
+
             function changeAddSubractButton(value) {
                 var addButton = $(addButtonSelector);
                 var subtractButton = $(subtractButtonSelector);
-                var hiddenClass = "hidden";
 
                 if (value >= 0) {
-                    addButton.removeClass(hiddenClass);
-                    subtractButton.addClass(hiddenClass);
+                    addButton.removeClass('hidden');
```

Comment:
Why did you remove the variable?

Reply:
because this is the only place where a variable is used like this, and I think is more readable to not use a variable for html property or class

Reply:
I would use the variable still, but its up to preference, I guess.

Reply:
Using the string is more expressive here. `hidden` is common enough class`